core/modules/czar/CzarConfig.cc

```diff
@@ -38,6 +41,22 @@ namespace {
 
 LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.CzarConfig");
 
+void QservLogger(struct timeval const& mtime,
```

Comment:
Ah, which function are we talking about? QservLogger? Well, if you look closely they are not quite the same as one looks for a server context while the other assume a client context and the messages reflect that. I suppose I could come up with some fancy way of distinguishing that but it would be straightforward. Is it really that annoying that an eight line function is "duplicated" in two places?